mcrouter/scripts/docker/container/ubuntu-packaged/usr/local/bin/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -x
+
+
+function write_default_config() {
+cat << EOF > /etc/mcrouter.json
+{
+  "pools": {
+    "A": {
+      "servers": [
+        "127.0.0.1:5001"
+      ]
+    }
+  },
+  "route": "PoolRoute|A"
+}
+EOF
+}
+
+function run_mcrouter() {
+        local PORT=$1
+        mcrouter \
+        --config file:/etc/mcrouter.json \
+        -p "${PORT}"
+}
+
+
+if [ -z "${MCROUTER_LISTEN_PORT}" ]; then
+  echo "MCROUTER_LISTEN_PORT unset! Defaulting to port 5000 ..."
+  export LISTEN_PORT=5000
```

Comment:
If you don't specify a port, does mcrouter just use it's own default port? I'd like to avoid re-defining a default if it's not needed.

Reply:
Unfortunately, if you don't specify the port, mcrouter exits 1,

```
+ mcrouter --config file:/etc/mcrouter.json
E0222 17:55:39.566481     6 StandaloneUtils.cpp:124] invalid ports
39.0.0-master mcrouter
usage:  [options] -p port(s) --config file:<config-file>
```